cockatrice/src/filtertree.cpp

```diff
@@ -295,6 +295,27 @@ bool FilterItem::acceptRarity(const CardInfoPtr info) const
     return false;
 }
 
+bool FilterItem::relationCheck(int cardInfo) const
+{
+    bool result, conversion;
+
+    result = (cardInfo == term.toInt(&conversion));
+    if (!conversion)
+    {
+        int termInt = term.mid(1).toInt();
+        if (term.startsWith('<')) {
+            result = (cardInfo < termInt);
+        }
+        else if (term.startsWith('>')) {
+            result = (cardInfo > termInt);
+        }
+        else if (term.startsWith('=')) {
+            result = (cardInfo == termInt);
+        }
```

Comment:
We don't seem to have a default case here, which is a bit troublesome. Maybe default it to ==?